typings/sharify.d.ts

```diff
@@ -22,6 +22,7 @@ declare module "sharify" {
       readonly ARTIST_COLLECTIONS_RAIL_IDS: string[]
       readonly CLIENT_NAVIGATION_V4: "experiment" | "control" // TODO: Remove after A/B test.
       readonly CMS_URL: string
+      readonly CDN_URL: string
```

Comment:
Is this related to this work, or did it come from somewhere else?

Reply:
I added `keyof GlobalData` to our `getENV` helper for better autocomplete. (`getENV` is a util for plucking isomorphic ENV vars -- `process.env` on the server, and `sd` on the client.) After doing that it revealed a type error which pointed to this missing value.